cmd/snap/cmd_help.go

```diff
@@ -20,21 +20,50 @@
 package main
 
 import (
+	"bytes"
+	"fmt"
 	"os"
 
 	"github.com/snapcore/snapd/i18n"
 
 	"github.com/jessevdk/go-flags"
 )
 
-var shortHelpHelp = i18n.G("Help")
+var shortHelpHelp = i18n.G("Show help about a command")
 var longHelpHelp = i18n.G(`
-The help command shows helpful information. Unlike this. ;-)
```

Comment:
Thanks for fixing this, I had no idea we had this in our actual code.

Reply:
Is this even reachable? "snap --help" and "snap help" both show the general command listing, which is nice.

Reply:
it's in the manpage (which is currently almost a showcase of bugs in go-flags, but it's better than nothing):
![image](https://user-images.githubusercontent.com/1543430/37366708-1c1b5dc0-26f9-11e8-9cf3-f5da68b7cfe2.png)